test/check/check.go

```diff
@@ -377,6 +402,16 @@ func checkForSecurityContextConstraintsRemoval(name string) error {
 	return isNotFound("SecurityContextConstraints", name, err)
 }
 
+func checkForSecretRemoval(name string) error {
+	err := framework.Global.Client.Get(context.Background(), types.NamespacedName{Name: name,Namespace: components.Namespace}, &corev1.Secret{})
```

Comment:
You can use `checkForSecret` here.

Reply:
I will change all `checkFor*Removal` function to use `checkFor*` in another PR

Reply:
fair enough